read-the-docs/source/Redmine.rst

```diff
@@ -30,6 +30,20 @@ Methods
 -  `add_attachment() <#add_attachment>`__
 
 
+get_ticket_content()
+--------------------
+
+``get_ticket_content(self, ticket_id=None)``
+
+Queries the Redmine API to get ticket_content using ticket_id. The
+ticket_content is expressed in a form of dictionary as a result of Redmine API's
+get issue: http://www.redmine.org/projects/redmine/wiki/Rest_Issues
```

Comment:
This link should be: http://www.redmine.org/projects/redmine/wiki/Rest_Issues#Showing-an-issue